filament/backend/src/vulkan/PlatformVkLinux.cpp

```diff
@@ -48,7 +48,7 @@ struct X11Functions {
 
 Driver* PlatformVkLinux::createDriver(void* const sharedContext) noexcept {
     ASSERT_PRECONDITION(sharedContext == nullptr, "Vulkan does not support shared contexts.");
-    static const char* requestedExtensions[] = {
+    static constexpr char* requestedExtensions[] = {
```

Comment:
I wouldn't use `static` inside a function, just to not run the risk that the compiler generates extra code for initialization on first call.